src/integration-test/java/apptest/CreateProcessIT.java

```diff
@@ -0,0 +1,146 @@
+package apptest;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import se.sundsvall.dept44.test.annotation.wiremock.WireMockAppTestSuite;
+import se.sundsvall.parkingpermit.Application;
+import se.sundsvall.parkingpermit.api.model.StartProcessResponse;
+import se.sundsvall.parkingpermit.integration.camunda.CamundaClient;
+
+import java.time.Duration;
+
+import static generated.se.sundsvall.camunda.HistoricProcessInstanceDto.StateEnum.ACTIVE;
+import static generated.se.sundsvall.camunda.HistoricProcessInstanceDto.StateEnum.COMPLETED;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.awaitility.Awaitility.setDefaultPollDelay;
+import static org.awaitility.Awaitility.setDefaultPollInterval;
+import static org.awaitility.Awaitility.setDefaultTimeout;
+import static org.hamcrest.Matchers.equalTo;
+import static org.springframework.http.HttpMethod.POST;
+import static org.springframework.http.HttpStatus.ACCEPTED;
+
+@WireMockAppTestSuite(files = "classpath:/CreateProcess/", classes = Application.class)
+class CreateProcessIT extends AbstractCamundaAppTest {
+
+	@Autowired
+	private CamundaClient camundaClient;
+
+	@BeforeEach
+	void setup() {
+		setDefaultPollInterval(500, MILLISECONDS);
+		setDefaultPollDelay(Duration.ZERO);
+		setDefaultTimeout(Duration.ofSeconds(30));
+
+		await()
```

Comment:
Check all `await()` and add `.atMost(XX, timeunit)` as the test will continue forever if no upper limit is set. In some places the upper limit has been set to 30 seconds.